.gitignore

```diff
@@ -41,3 +41,5 @@ lib*.pc
 /env/
 /GIT-VERSION-FILE
 /version
+/tests/schema-filter
+.vscode/
```

Comment:
Please refrain from adding personal preferences to the repository, add .vscode to your local .gitignore setup!